src/Reducer.js

```diff
@@ -1,7 +1,7 @@
 import sets from './json/sets.json';
 import reducers from './actions';
 
-const initialState = {setList: sets, cardList: []}  
+const initialState = {setList: sets, cardList: [], jwt: {}, isUserLoggedIn: false};  
```

Comment:
isn't JWT going to be a string?

Reply:
I thought that we discussed it returning the token and the username?  I assumed it would be an object.  Easily changed based on what the backend returns.